src/Symfony/Component/HttpKernel/BootingKernel.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel;
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\Exception\BadMethodCallException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Bundle\BootingBundle;
+use Symfony\Component\HttpKernel\Bundle\BundleInterface;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class BootingKernel implements KernelInterface
```

Comment:
Why not, but...

I was wondering if we should explicit the kernel state at this point in the class name. I finally tend to say yes, because it's also a unusable kernel instance. I.e you cannot use `handle()`. So `ImmutableKernel` is true, but not enough.

What do you think ?